### PR TITLE
Use https in Employee not http

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Use https
| Type?         |  improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9136)

### CODE REVIEW

**Review by mickaelandrieu — DISMISSED** [review]

File: classes/Employee.php [hunk 1]
```diff
@@ -507,10 +507,9 @@ public function logout()
     public function favoriteModulesList()
     {
         return Db::getInstance()->executeS('
-			SELECT `module`
-			FROM `'._DB_PREFIX_.'module_preference`
-			WHERE `id_employee` = '.(int)$this->id.' AND `favorite` = 1 AND (`interest` = 1 OR `interest` IS NULL)'
-        );
+		SELECT `module`
```
Comment (PierreRambaud) [inline classes/Employee.php]: Why did you change the indentation?

Comment (ghost) [reply classes/Employee.php]: @PierreRambaud 

It's my sublime pluggin but we found 3 idententation in this class : 

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L304

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L381

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L507

Tell me which is the right one

Comment (PierreRambaud) [reply classes/Employee.php]: This one look really good, https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L304
It's PSR2 / Symfony style compliant :)

**Comment by mickaelandrieu** [comment]
"Make ident grat again" => ahahah don't worry we can change the git label name :)

Thansk @okom3pom

**Comment by ghost** [comment]
oh !

**Review by mickaelandrieu — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
and merged!

Thanks @okom3pom
